scripts/examples/typescript/README.md

```diff
@@ -0,0 +1,41 @@
+# A Typescript bindings for XenAPI
+A Typescript bindings for XenAPI, which is inspired by [Python XenAPI](https://xapi-project.github.io/xen-api/usage.html).
+
+## Usage
+Install lib from npm repository
+```
+$ npm install xen-api-ts
+```
+
+The usage of Typescript XenAPI is almost identical to the Python XenAPI, except it's asynchronous and requires async/await.
```

Comment:
Let's link to the python library for easier navigation

also
```suggestion
The usage of Typescript XenAPI npm library is almost identical to the Python XenAPI library, except it's asynchronous and requires async/await.
```

Reply:
I've linked https://xapi-project.github.io/xen-api/usage.html at line 2